src/meshes/t8code_mesh.jl

```diff
@@ -141,10 +171,19 @@ function T8codeMesh(trees_per_dimension; polydeg,
         periodicity = Tuple(periodicity)
     end
 
-    conn = T8code.Libt8.p4est_connectivity_new_brick(trees_per_dimension..., periodicity...)
-    do_partition = 0
-    cmesh = t8_cmesh_new_from_p4est(conn, mpi_comm(), do_partition)
-    T8code.Libt8.p4est_connectivity_destroy(conn)
+    if NDIMS == 2
+        conn = T8code.Libt8.p4est_connectivity_new_brick(trees_per_dimension...,
+                                                         periodicity...)
+        do_partition = 0
+        cmesh = t8_cmesh_new_from_p4est(conn, mpi_comm(), do_partition)
+        T8code.Libt8.p4est_connectivity_destroy(conn)
+    elseif NDIMS == 3
+        conn = T8code.Libt8.p8est_connectivity_new_brick(trees_per_dimension...,
+                                                         periodicity...)
+        do_partition = 0
```

Comment:
```suggestion
    do_partition = 0
    if NDIMS == 2
        conn = T8code.Libt8.p4est_connectivity_new_brick(trees_per_dimension...,
                                                         periodicity...)
        cmesh = t8_cmesh_new_from_p4est(conn, mpi_comm(), do_partition)
        T8code.Libt8.p4est_connectivity_destroy(conn)
    elseif NDIMS == 3
        conn = T8code.Libt8.p8est_connectivity_new_brick(trees_per_dimension...,
                                                         periodicity...)
```

I guess you could shorten the code here a bit.